extensions/grapher/pom.xml

```diff
@@ -17,17 +17,14 @@
     <dependency>
       <groupId>com.google.inject.extensions</groupId>
       <artifactId>guice-assistedinject</artifactId>
-      <version>${project.version}</version>
```

Comment:
All these extension versions were moved to the parent pom. They will eventually be used in the bom